### DISCUSSION TITLE
How to embed OpenDJ

### DISCUSSION THREAD

**Discussion by AnthoDAvey** [OpenIdentityPlatform/OpenDJ#339] [Q&A] [answered]
Hello, there are very few examples about how to embed OpenDJ but all seem to be outdated as it uses the not anymore supported version from forge rock.

So far I've added the dependencies described here: https://mvnrepository.com/artifact/org.openidentityplatform.opendj/opendj-server/4.6.3

Is there any up to date documentation or example about how to proceed form here?

Additional info here, this is not for testing purpose, the software I'm working on requires to manage users/group and we have chosen to use OpenDJ to do so.

**Comment by vharseko** [comment, marked as answer]
please check https://github.com/OpenIdentityPlatform/OpenDJ/blob/55f9c10c4ffe6a0a3979a56eea5c47430cd6500d/opendj-server-legacy/src/main/java/org/forgerock/opendj/server/embedded/EmbeddedDirectoryServer.java

**Reply by AnthoDAvey** [reply]
Hello Valery, thanks for the quick answer.

**Comment by vharseko** [comment]
https://repo1.maven.org/maven2/org/openidentityplatform/openam/embedded.opendj/15.0.0.1/embedded.opendj-15.0.0.1-sources.jar

**Reply by vharseko** [reply]
[How to Run OpenDJ based Embedded LDAP in Spring Boot Application](https://github.com/OpenIdentityPlatform/OpenDJ/wiki/How-to-Run-OpenDJ-based-Embedded-LDAP-in-Spring-Boot-Application)
